torch/_inductor/quantized_lowerings.py

```diff
@@ -144,11 +157,19 @@ def int4pack_mm_cpu(
                 (mat1, mat2, group_size, qScaleAndZeros), aten_layout
             ).output_node()
 
+        # define functions to generate example inputs for weight and group size
+        # otherwise, autotuner generates example inputs of all zeros for them
+        input_gen_fns = {
+            1: lambda x: V.graph.constants[x.get_name()],  # packed weight
```

Comment:
How do we know the input is a graph constant?  What if it is dynamic?

Reply:
Maybe we can also add a check in the lowering to use GEMM Template only when weight is a constant buffer.

Reply:
Thanks. I have modified this part so that we don't have to assume that weight is a constant buffer.

Reply:
We should not assume weights are constants, people use pytorch is weird ways.

Reply:
Thanks.